pages/wishlist.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Snigdha Kamal" />
+    <meta name="description" content="" />
+    <meta name="tags" content="" />
```

Comment:
You can fill these meta values with description about your web application

Reply:
Good catch, will do!